providers/implementations/exchange/build.info

```diff
@@ -21,4 +22,6 @@ ENDIF
 IF[{- !$disabled{ec} -}]
   SOURCE[$ECX_GOAL]=ecx_exch.c
   DEFINE[$ECX_GOAL]=$ECDEF
+  SOURCE[$ECDH_GOAL]=ecdh_exch.c
+  DEFINE[$ECDH_GOAL]=$ECDEF
```

Comment:
@mattcaswell is this `DEFINE` superfluous?

Reply:
I think it is. Should we ever split things up per algorithm, the def variable should be a separate one

Reply:
Done: removed the second `DEFINE`